pkg/controller/actuator.go

```diff
@@ -195,9 +229,27 @@ func (a *actuator) createSeedResources(ctx context.Context, ex *extensionsv1alph
 	return a.createManagedResource(ctx, namespace, v1alpha1.CertManagementResourceNameSeed, "seed", renderer, v1alpha1.CertManagementChartNameSeed, certManagementConfig, nil)
 }
 
-func (a *actuator) createShootResources(ctx context.Context, cluster *controller.Cluster, namespace string) error {
+func (a *actuator) createShootResources(ctx context.Context, ex *extensionsv1alpha1.Extension, cluster *controller.Cluster, namespace string) error {
```

Comment:
What do you think about passing the `CertConfig` here instead of `ex *extensionsv1alpha1.Extension` and also make the `createSeedResource` accept the `CertConfig`? This saves us some efforts because we only need to decode once.

Reply:
good point

Reply:
done